core/templates/pages/exploration-player-page/exploration-player-page.module.ts

```diff
@@ -35,24 +35,39 @@ import { SwitchContentLanguageRefreshRequiredModalComponent } from
   // eslint-disable-next-line max-len
   'pages/exploration-player-page/switch-content-language-refresh-required-modal.component';
 import { InteractionExtensionsModule } from 'interactions/interactions.module';
+import { LearnerLocalNavComponent } from './layout-directives/learner-local-nav.component';
+import { FlagExplorationModalComponent } from './modals/flag-exploration-modal.component';
+import { FeedbackPopupComponent } from './layout-directives/feedback-popup.component';
+import { MatButtonModule } from '@angular/material/button';
+import { ExplorationSuccessfullyFlaggedModalComponent } from './modals/exploration-successfully-flagged-modal.component';
 
 @NgModule({
   imports: [
     BrowserModule,
     HttpClientModule,
     InteractionExtensionsModule,
     NgbModalModule,
-    SharedComponentsModule
+    MatButtonModule,
```

Comment:
MatButtonModule is already present in material.module.ts, right?

Reply:
done